hex/helpers/lib/run.exs

```diff
@@ -118,7 +118,7 @@ defmodule DependencyHelper do
   end
 
   defp fetch_public_key(repo, repo_url, auth_key, fingerprint) do
-    case Hex.Repo.get_public_key(repo_url, auth_key) do
+    case Hex.Repo.get_public_key(%{trusted: true, url: repo_url, auth_key: auth_key}) do
```

Comment:
What does `trusted: true` mean?

I see these params are used similarly here: https://github.com/hexpm/hex/blob/1881f9fe8e0571ba7fdcfc86ecf484913125dc37/lib/mix/tasks/hex.repo.ex#L218-L220

I'm not an Erlang/Hex expert, but typically in a package manager an arg like "trusted" implies a private/insecure registry of some kind...

Reply:
I _think_ Elixir will pattern match this, so `%{trusted: false` would cause this to not match.

Reply:
Trusted isn’t used in a clear way. I will need more time to dissect it in the code to give a better answer.

Reply:
`trusted: true` appears to indicate that there is a fingerprint that is being used to verify the key.

Reply:
Since we are using a fingerprint to verify the key, that should mean that our use of the `trusted` key here matches the intended use in the code.